tests/AssertTest.php

```diff
@@ -289,8 +290,18 @@ public function getTests()
             array('uuid', array('ff6f8cb0-c57da-51e1-9b21-0800200c9a66'), false),
             array('uuid', array('af6f8cb-c57d-11e1-9b21-0800200c9a66'), false),
             array('uuid', array('3f6f8cb0-c57d-11e1-9b21-0800200c9a6'), false),
-
+            array('throws', array(function() { throw new LogicException('test'); }, 'LogicException'), true),
+            array('throws', array(function() { throw new LogicException('test'); }, 'IllogicException'), false),
+            array('throws', array(function() { throw new Exception('test'); }), true),
         );
+        if (interface_exists('Throwable')) {
```

Comment:
I believe so. It is more explicit. We want to run this test in PHP7. We do not care if Throwable exists or not. =)
What if I write a test that need scalar type hints, what should I feature check that against? Do I need another if statement for that test?

Since this is a data provider we need this special if-statement. Normally we would annotate the test with `@requires PHP 7.0`